### PR TITLE
Refactorama Figgy Molecularization

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: roles/pulibrary.figgy/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+Role Name
+=========
+
+A brief description of the role goes here.
```
Comment (carolyncole) [inline roles/pulibrary.figgy/README.md:4]: We should add a description of figgy here

File: roles/pulibrary.figgy/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Requirements
+------------
+
+Any pre-requisites that may not be covered by Ansible itself or the role should
```
Comment (carolyncole) [inline roles/pulibrary.figgy/README.md:9]: Did we want to describe the additional items added in the playbook?

File: roles/pulibrary.figgy/molecule/default/tests/test_default.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']
+).get_hosts('all')
+
+
+def test_hosts_file(host):
```
Comment (carolyncole) [inline roles/pulibrary.figgy/molecule/default/tests/test_default.py:10]: We should probably add a couple tests, but that would not need to be done here...

File: roles/pulibrary.figgy/README.md (hunk 1 above)
Comment (carolyncole) [inline roles/pulibrary.figgy/README.md:4]: We should add a description of figgy here

File: roles/pulibrary.figgy/README.md (hunk 2 above)
Comment (carolyncole) [inline roles/pulibrary.figgy/README.md:9]: Did we want to describe the additional items added in the playbooks for workers ect.?

**Review by carolyncole — CHANGES_REQUESTED** [review]
Looking good.  I think we should at the very least update the Readme before a merge.

File: roles/pulibrary.figgy/molecule/default/tests/test_default.py (hunk 3 above)
Comment (kayiwa) [reply roles/pulibrary.figgy/molecule/default/tests/test_default.py]: added some tests.

**Review by carolyncole — APPROVED** [review]

**Pull request merged**